unit10.js

```diff
@@ -0,0 +1,127 @@
+// Bai1 Phep nhan
+const calcMultipliers = n => {
+    for (let i=1; i<=10; i++) {
+        console.log(`${n} x ${i} = ${n*i}`)
+    }
+}
+
+// Bai2 So chan
+const getEvenNumber = n => {
+    if (n >= 1 && n <= 30) {
+        for (let i=1; i<=(n/2); i++) {
```

Comment:
Chú ý dấu cách
```suggestion
        for (let i=1; i <= (n/2); i++) {
```
Nhìn OK 👍